docs/guide/piece-pages.md

```diff
@@ -1,257 +1,162 @@
+## Creating a piece page module
 
-In general piece modules are concerned with *editing and APIs*, while piece page modules are concerned with *browsing as part of a website*. For developers familiar with the "model / view / controller" pattern: **piece modules are the model layer for your content type, while piece page modules are the view layer.**
+A common use for these is a **blog**. The related piece type in that case might come from the `article` piece module (refer to [the pieces guide](/guide/pieces.md#creating-a-piece-type) for more on that). **The piece page would then be a module named `article-page`.**
 
-Some projects just need a widget module for each piece, but most will want to let users view a page for each one, or at least browse and paginate through a complete list. That's where piece pages shine.
+```javascript
+// modules/article-page/index.js
+module.exports = {
+  extend: '@apostrophecms/piece-page-type',
+  options: {
+    label: 'Articles Page'
+  }
+};
+```
 
-## Piece Pages by Example
+You would **add this to the `app.js` configuration** [like any other module](/guide/modules.html#setting-up-a-module). You could also add a field schema or other features as on page type modules.
 
-In A3, piece pages work much like 2.x, with a few important changes. Most of those changes are described by the the [new module format](module-format-example.md).
+::: tip
+Piece page modules are not required to be named using the piece name plus `-page` suffix. If they are named like that the associated piece type will be found automatically. If you want to name it different, or if you need more than one piece page type for a single piece type, use the [`piecesModuleName` option](/reference/module-api/module-options.md#piecemodulename) to identify the correct piece type.
+:::
 
-Just like in A2, piece page modules and piece modules come in pairs. And the name of the piece page module automatically determines which piece module it works together with. So we don't need very much code in our actual module in order to get started.
+Instead of a single `page.html` template, this module gets two templates.
 
-Here's an example of a piece page module that works with the product piece described in the [pieces section](pieces.md):
+| Template file name | What is it? |
+| ------------------ | ----------- |
+| `index.html` | Template for listing pieces using `data.pieces` (the **"index page**) |
+| `show.html` | Template to display individual piece information using `data.piece` (a **"show page"**) |
 
-```js
-// app.js
-require('apostrophe')({
-  modules: {
-    // ... other modules ...
-    // piece module
-    product: {},
-    // piece page module
-    'product-page': {}
-  }
-});
-```
+Before we go any further, make sure you have reviewed the [page type guide](/guide/pages.md). Everything there also applies to index pages and templating is mostly the same for show pages as well.
```

Comment:
"make sure you have reviewed" sounds a little 'dad' heavy. maybe we could try a fun